schema/schema_test.go

```diff
@@ -113,6 +113,36 @@ var _ = Describe("Schema", func() {
 		})
 	})
 
+	Describe("SchemaPaths", func() {
+		var metadataIdSchema *Schema
+
+		BeforeEach(func() {
+			var idExists bool
+			manager := GetManager()
+			schemaPath := "../tests/test_schema_metadata.yaml"
+			Expect(manager.LoadSchemaFromFile(schemaPath)).To(Succeed())
+			metadataIdSchema, idExists = manager.Schema("metadata_id")
+			Expect(idExists).To(BeTrue())
+		})
+
+		It("GetSchemaByTemplatePath", func() {
+			data := map[string]interface{}{
+				"id": "1234",
+			}
+			path, err := metadataIdSchema.GenerateCustomPath(data)
+			Expect(err).To(Succeed())
+			Expect(metadataIdSchema).To(Equal(GetSchemaByPath(path)))
```

Comment:
If I understand the intention of the test, his should be like
```
Expect(GetSchemaByPath(path)).To(Equal(metadataIdSchema))
```
(`Expect(ACTUAL).To(Equal(EXPECTED))`(https://onsi.github.io/gomega/#making-assertions))